kodilla-hibernate/src/main/java/com/kodilla/hibernate/invoice/Item.java

```diff
@@ -0,0 +1,81 @@
+package com.kodilla.hibernate.invoice;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "ITEMS")
+public class Item {
+
+    private int id;
+    private Product product;
+    private BigDecimal price;
+    private int quantity;
+    private BigDecimal value;
+    private Invoice invoice;
+
+    public Item() {
+    }
+
+    public Item(Product product, BigDecimal price, int quantity) {
+        this.product = product;
+        this.price = price;
+        this.quantity = quantity;
+        this.value = price.multiply(BigDecimal.valueOf(quantity));
+    }
+
+    @Id
+    @GeneratedValue
+    public int getId() {
+        return id;
+    }
+
```

Comment:
Co oznacza cascade all? Jak usuniesz item, to usuniesz tez produkt? Raczej cascade.persist